js/everything.js

```diff
@@ -218,6 +227,8 @@ var ZenPlayer = {
                     }
                     window.sessionStorage[videoID] = resumeTime;
                 }
+                var updatedUrl = that.videoUrl + "&t=" + parseInt(resumeTime, 10);
```

Comment:
Why is `parseInt()` used here? Won't `resumeTime` always be a `Number`?

Also, these 2 lines should be surrounded in an `if (resumeTime > 0)` block - there's no point in updating the URL if we're just adding `&t=0`